frontend/src/modules/app/ToolBar.tsx

```diff
@@ -248,6 +299,9 @@ export const ToolBar = ({
     initParameterUpdate();
   }, [ventilating, initParameterUpdate]);
```

Comment:
What is the purpose of this `useEffect`? Why are we calling `initParameterUpdate` from a `useEffect` rather than from the `Start Ventilation` button's click handler?

Reply:
I am not really sure why I have done it this way & rather it would be done directly through `Start Ventilation` button's click handler. I assume It would be something to do with functionality `isVentilatorOn` local state. But I believe its better to refactor Toolbar removing this as it seems to have lot of dependency.